committeeoversightapp/templates/committeeoversightapp/committee_detail_page.html

```diff
@@ -1,32 +1,167 @@
 {% extends "base.html" %}
+{% load static %}
+
+{% block extra_js %}
+  <script src="https://code.highcharts.com/highcharts.js"></script>
+  <script src="{% static 'js/chart_helper.js' %}"></script>
+{% endblock %}
 
 {% block content %}
-  <div class="col-12 text-center">
-    <h2 class="mb-4">{{ page.title }}</h2>
-    <p class="lead font-weight-light text-dark"><strong>Chairperson:</strong> John Smith</p>
+  <div class="row justify-content-center">
+    <div class="col-12 col-md-10 text-center">
+      <h2 class="my-2">{{ page.title }}</h2>
```

Comment:
The mockup includes the name of the project at the top of the page, but I'd like to ask Lugar about coming up with text/a logo that can fit into the top navbar instead of manually including the name on each page. I'm keeping track of those questions here: #107